### PR TITLE
refactor(final_step): add code climate badge for maintainability

### PR DESCRIPTION
**Pull request by AlexMomot-717**

### CODE REVIEW

File: page_analyzer/app.py [hunk 1]
```diff
@@ -22,32 +22,33 @@
 
 @app.route("/")
 def index() -> str:
-    return render_template("index.html")
+    url_name = ""
+    return render_template("index.html", url_name=url_name)
```
Comment (svntmr) [inline page_analyzer/app.py]: what is the reason for this change?

File: page_analyzer/templates/base.html [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
   <body class="min-vh-100 d-flex flex-column">
     <header class="flex-shrink-0">
       <nav class="navbar navbar-expand-md navbar-dark bg-dark px-3">
         <a class="navbar-brand" href="/">Анализатор страниц</a>
         <div id="navbarNav">
           <ul class="navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link" href="{{ url_for('list_urls') }}">Сайты</a>
-            </li>
+            <li class="nav-item"><a class="nav-link" href="/urls">Сайты</a></li>
```
Comment (svntmr) [inline page_analyzer/templates/base.html]: is it possible to return `url_for` back?

File: page_analyzer/templates/url.html [hunk 3]
```diff
@@ -26,7 +26,7 @@ <h1>Сайт: {{ url.name }}</h1>
     </div>
 
     <h2 class="mt-5 mb-3">Проверки</h2>
-    <form method="post" action="{{ url_for('add_url_check', url_id=url.id) }}">
+    <form method="post" action="/urls/{{ url.id }}/checks">
```
Comment (svntmr) [inline page_analyzer/templates/url.html]: is it possible to return `url_for` back?

File: page_analyzer/templates/urls.html [hunk 4]
```diff
@@ -21,9 +21,7 @@ <h1>Сайты</h1>
           <tr>
             <td>{{ url.url_id }}</td>
             <td>
-              <a href="{{ url_for('show_url', url_id=url['url_id']) }}"
-                >{{ url.url_name }}</a
-              >
+              <a href="/urls/{{ url.url_id }}">{{ url.url_name }}</a>
```
Comment (svntmr) [inline page_analyzer/templates/urls.html]: is it possible to return url_for back?

**Review by svntmr — CHANGES_REQUESTED** [review]
looks like we can remove some debug-related changes

File: page_analyzer/app.py (hunk 1 above)
Comment (AlexMomot-717) [reply page_analyzer/app.py]: the reason is to keep incorrect url value in the url input field

Comment (AlexMomot-717) [reply page_analyzer/app.py]: Снимок экрана от 2024-02-20 08-45-50

Comment (svntmr) [reply page_analyzer/app.py]: wdyt, does it make sense not to pass it here and fetch value for this field from the flash messages in the view instead? or make it optional - so you'll pass it only when needed

**Review by svntmr — APPROVED** [review]
one small thing that can be improved, but LGTM otherwise!

**Pull request merged**